build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
   id 'org.sonarqube' version '2.7.1'
   id "com.diffplug.gradle.spotless" version "3.27.1"
+  id "jp.skypencil.spotbugs.snom" version "0.4.13"
```

Comment:
What is this?

Reply:
A gradle plugin implemented from scratch. Will be official spotbugs-gradle-plugin 4.0.0.
refs https://github.com/spotbugs/spotbugs-gradle-plugin/issues/180